examples/sklearn_elasticnet_wine/conda.yaml

```diff
@@ -2,6 +2,7 @@ name: tutorial
 channels:
   - defaults
 dependencies:
+  - cloudpickle
```

Comment:
Nit: Can you pin the version of cloudpickle to something compatible with Python 3.6 like `0.6.1`?

Reply:
completed pinned cloudpickle=0.6.1